projects/igniteui-angular/src/lib/grids/grid/grid.component.html

```diff
@@ -123,7 +123,7 @@
     <div class="igx-grid__tfoot-thumb" [hidden]='!hasVerticalSroll()' [style.height.px]='summariesHeight' [style.width.px]="scrollWidth"></div>
 </div>
 
-<div class="igx-grid__scroll" [style.height]="'18px'" #scr [hidden]="unpinnedWidth - totalWidth >= 0">
+<div class="igx-grid__scroll" [style.height]="'18px'" #scr [hidden]="unpinnedWidth - (totalWidth - 18) >= 0">
```

Comment:
This seems to allow the vertical scrollbar to cover the last cells right edge before a horizontal scrollbar is shown. Why is this part of this PR?

I believe we moved the v scrollbar out of the tbody so that it never covers the cells.

Reply:
@ChronosSF There were failing tests. The issues was with the horizontal scroll that opened up a gap on the bottom. I fixed it by clearing the cache when the columns get recalculated.
Note: The issue is when the columns don't have widths set.